public/Lab-Files/Lesson03/Assignments/blog-page.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="blog-page.css">
+    <meta charset="UTF-8">
+    <title>Blog Page</title>
+</head>
+<body>
+    <Header>
```

Comment:
You should really use lowercase tags, this could have been written as `<header>` this way all the tags are consistent. Remember tags are not case sensitive but using lowercase tag names is the recommended practice. Keep up the good work!